bee-test/src/rand/bytes.rs

```diff
@@ -3,10 +3,12 @@
 
 use rand::Rng;
 
+/// Generates a vector of random bytes with a given length.
 pub fn rand_bytes(len: usize) -> Vec<u8> {
     (0..len).map(|_| rand::random::<u8>()).collect::<Vec<u8>>()
 }
```

Comment:
overlay friendly to the compiler: `(0..len).map(|_| rand::random()).collect()`

Reply:
Interesting that clippy doesn't say anything about that